backoffice/urls.py

```diff
@@ -28,4 +28,14 @@
         views.reject_billing_account,
         name="backoffice_reject_billing_account",
     ),
+    path(
+        "bookings/closebooking/<int:booking_id>/<int:should_lock>/",
```

Comment:
```suggestion
        "bookings/close/<int:booking_id>",
```
And then maybe a `lock=true` in the query params? Defaulting to `false` if not set?